build/charts/yorkie-cluster/templates/istio/ratelimit/service.yaml

```diff
@@ -0,0 +1,25 @@
+{{ if .Values.ratelimit.enabled }}
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ .Values.ratelimit.name }}
+  namespace: {{ .Values.yorkie.namespace }}
```

Comment:
> How about moving ratelimit resources to `istio` namespace? Since ratelimit is applied to ingressgateway in `istio` namespace, it will be better to place them in `istio` namespace.

Same here.